modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/Kaleodesigner.testcase

```diff
@@ -147,7 +147,6 @@ definition {
 
 	@priority = "5"
 	test AddJoinXORWorkflowDefinitionViaViewTab {
-		property portal.acceptance = "quarantine";
```

Comment:
It should not be removed, but replaced by `property portal.acceptance = "true";`